l10n_it_fatturapa_in/models/account.py

```diff
@@ -180,8 +180,14 @@ def set_einvoice_data(self, fattura):
         amount_total = float(
             fattura.DatiGenerali.DatiGeneraliDocumento.
             ImportoTotaleDocumento or 0.0)
+
         reference = fattura.DatiGenerali.DatiGeneraliDocumento.Numero
+        if not self.reference:
+            self.reference = reference
+
         date_invoice = fattura.DatiGenerali.DatiGeneraliDocumento.Data
+        if not self.date_invoice:
+            self.date_invoice = date_invoice
```

Comment:
Visto che set_einvoice_data viene chiamato anche dal wizard di import, dove questi 2 campi vengono già scritti in altro punto, forse sarebbe un po' più pulito mettere la scrittura di questi 2 campi in un metodo dedicato

Reply:
grazie @eLBati della review!
Ho messo la scrittura dei due campi nel metodo dedicato `set_vendor_bill_data` che viene chiamato sia durante l'import che durante il link, che ne dici?